src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -283,6 +283,22 @@ private ReadOnlyMemory<byte> GetPropertyRawValue(int valueIndex)
                 : JsonReaderHelper.TranscodeHelper(segment);
         }
 
+        internal ReadOnlySpan<byte> GetRawSpan(int index, JsonTokenType expectedType)
```

Comment:
It's that we're only passing a parameter just to use it in a debug assert. I would recommend renaming this to something like `GetRawStringValue(int index)` and simply add an assertion such as
```C#
Debug.Assert(row.TokenType is JsonTokenType.String or JsonTokenType.PropertyName);
```